worker/sshserver/mocks/system_state_mock.go

```diff
@@ -12,6 +12,7 @@ package mocks
 import (
```

Comment:
We don't use `mocks` packages as it's possible to use these else where, which causes coupled dependencies.